src/Command/ReformatCommand.php

```diff
@@ -0,0 +1,92 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Command;
+
+use PHPStan\Command\ErrorFormatter\ErrorFormatter;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ReformatCommand extends Command
+{
+
+	private const NAME = 'reformat';
+
+	public function __construct()
+	{
+		parent::__construct();
```

Comment:
This constructor doesn't have to be here.

Reply:
Removed